lib/mjsonwp/mjsonwp.js

```diff
@@ -245,6 +245,11 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
         driverRes = null;
       }
 
+      // if the status is not 0,  throw the appropriate error for status code.
+      if (util.hasValue(driverRes) && util.hasValue(driverRes.status) && parseInt(driverRes.status, 10) !== 0) {
+        throw errorFromCode(driverRes.status, driverRes.value);
```

Comment:
Sure, making the necessary changes to reflect the same

https://github.com/appium/appium-uiautomator2-server/pull/16

https://github.com/appium/appium-base-driver/pull/66